bot/cogs/snakes.py

```diff
@@ -2,10 +2,33 @@
 import logging
 from typing import Any, Dict
 
+from googleapiclient.discovery import build
 from discord.ext.commands import AutoShardedBot, Context, command
+from discord import Embed
+
+import aiohttp
+import json
```

Comment:
these imports aren't gonna pass linting. wrong import orders and groupings. you should fix them so they are PEP8 compliant. Why aren't you guys using flake8?